src/pixelator/plot/__init__.py

```diff
@@ -10,7 +10,17 @@
 from matplotlib.colors import Normalize
 import networkx as nx
 import pandas as pd
-from typing import Literal, Tuple
+from typing import Literal, Tuple, Optional
+from plotnine import (
```

Comment:
What is `plotnine`? Have we added that dependency?

Reply:
It's a library for plotting which mimics ggplot2 in R. I could also try to switch to a different library, but it makes things way easier in order to get similar output to what we get with pixelatorR.

Reply:
If this is ok, I'd like to add it as a dependency :-)

Reply:
https://github.com/PixelgenTechnologies/pixelator/blob/dev/DEVELOPERS.md#update-poetrylock